### PR TITLE
Update container-lifecycle-hooks.md

### PR DESCRIPTION
**Pull request by PulkitSinghDev**
**Description**
Just fixed some gramatical mistakes

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
Pls sign the CLA in order for the PR to be reviewed.
more info https://github.com/kubernetes/website/pull/29335#issuecomment-897056583

**Comment by PulkitSinghDev** [comment]
@Arhell already done 👍🏼 
image

**Comment by RA489** [comment]
> @Arhell already done 👍🏼
/check-cla

File: content/en/docs/concepts/containers/container-lifecycle-hooks.md [hunk 1]
```diff
@@ -37,9 +37,9 @@ No parameters are passed to the handler.
 `PreStop`
 
 This hook is called immediately before a container is terminated due to an API request or management
-event such as a liveness/startup probe failure, preemption, resource contention and others. A call
+events such as a liveness/startup probe failure, preemption, resource contention, and others. A call
 to the `PreStop` hook fails if the container is already in a terminated or completed state and the
-hook must complete before the TERM signal to stop the container can be sent. The Pod's termination
+hook must be complete before the TERM signal to stop the container can be sent. The Pod's termination
```
Comment (RinkiyaKeDad) [inline content/en/docs/concepts/containers/container-lifecycle-hooks.md:42]:
```suggestion
hook must complete before the TERM signal to stop the container can be sent. The Pod's termination
```
You can either leave it as it was or if you want to introduce `be` I believe the line would be

```suggestion
hook must be completed before the TERM signal to stop the container can be sent. The Pod's termination
```

File: content/en/docs/concepts/containers/container-lifecycle-hooks.md [hunk 2]
```diff
@@ -75,14 +75,14 @@ execution, the Pod's phase will be `Terminating` and remain there until the Pod
 both the `PreStop` hook to execute and for the Container to stop normally. If, for example,
 `terminationGracePeriodSeconds` is 60, and the hook takes 55 seconds to complete, and the Container
 takes 10 seconds to stop normally after receiving the signal, then the Container will be killed
-before it can stop normally, since `terminationGracePeriodSeconds` is less than the total time
+before it can stop normally since `terminationGracePeriodSeconds` is less than the total time
```
Comment (RinkiyaKeDad) [inline content/en/docs/concepts/containers/container-lifecycle-hooks.md:78]: I'm not sure if removing the `,` here is the right choice. Could you please let me know your reasoning?

**Review by RinkiyaKeDad — COMMENTED** [review]
Thanks for the PR! Pointed out a couple of nits :)

**Pull request closed without merging**